utils/version.go

```diff
@@ -0,0 +1,26 @@
+package utils
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/Masterminds/semver"
+)
+
+func CheckVersion(name string, version string, minVersion string) error {
+	// strip lnd commit
+	version = strings.Split(version, " ")[0]
+	// strip cln rc
+	version = strings.Split(version, "rc")[0]
+	version = strings.Split(version, "-")[0]
+
+	parsed, err := semver.NewVersion(version)
+	if err != nil {
+		return fmt.Errorf("Could not parse %s version %s: %s", name, version, err.Error())
+	}
+
+	if parsed.LessThan(semver.MustParse(minVersion)) {
```

Comment:
`MustParse` panics on error. Not a fan. Could we use the function that returns an error and handle it gracefully?